test/fixtures/experimental/uncategorised/33/actual.js

```diff
@@ -1 +0,0 @@
-@foo class Foo {}
```

Comment:
Where'd this test go?

Reply:
It was almost the same as `decorators/class-decorator/actual.js`.  https://github.com/babel/babylon/blob/bfd9c7ba508cf06813636e2dc6adf5658c743b34/test/fixtures/experimental/decorators/class-decorator/actual.js#L1-L4

Reply:
I think we can keep both, since in `decorators/class-decorator/actual.js` the decorator is on a different line and thus have different locations.

Reply:
Alright, I added it back.

Reply:
It is `same-line`

Reply:
I am **so** glad that I have other people check my work. Sorry about that 😳.